Fix invisible buttons in password fields (desktop Safari)

Iksas opened the pull request:
**By submitting this pull request, I confirm the following:**

- [X] I have read and understood the [contributors guide](https://github.com/pi-hole/AdminLTE/blob/master/CONTRIBUTING.md), as well as this entire template.
- [X] I have made only one major change in my proposed changes.
- [X] I have commented my proposed changes within the code.
- [X] I have tested my proposed changes.
- [X] I am willing to help maintain this change if there are issues with it later.
- [X] I give this submission freely and claim no ownership.
- [X] It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
- [X] I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))
- [X] I have Signed Off all commits. ([`git commit --signoff`](https://git-scm.com/docs/git-commit#Documentation/git-commit.txt---signoff))

---

**What does this PR aim to accomplish?:**

When using dark themes in desktop Safari, the autofill button cannot be seen because it's colored black on a black background. For the same reason, the caps lock indicator cannot be seen. This PR fixes both issues.

**How does this PR accomplish the above?:**

The issues are fixed by using the text color for the autofill button, and by inverting the caps lock indicator:

<img width="473" alt="dark" src="https://user-images.githubusercontent.com/17005217/147355714-ae845e04-f7fd-4068-8075-d5845a49a4c8.png">
<img width="474" alt="darker" src="https://user-images.githubusercontent.com/17005217/147355719-59edcae4-fb62-44a7-b482-cc01c897018c.png">
<img width="501" alt="lcars" src="https://user-images.githubusercontent.com/17005217/147355725-e108e781-1605-42ca-86d1-2fd62aaa905b.png">


**What documentation changes (if any) are needed to support this PR?:**

none

Iksas committed:
fix invisible buttons in password fields

Signed-off-by: Iksas <Iksas@users.noreply.github.com>

yubiuser commented:
What's about the other themes? Do they offer an caps-lock indicator that needs to be fixed? 

Are the indicators browser-dependent? I haven't seen them on Chrome or Firefox?

Iksas commented:
You're right, all dark themes are affected. Don't know how I missed this, I have now fixed them all. The color of the password button is now the same as the text color.

The indicators are indeed browser-dependent. Desktop Safari is the only browser that I know of that has them.

Also, please let me know if I have added the fixes in the right lines. It was a bit hard to decide where to put them in default-dark.css
